### DISCUSSION TITLE
Notes categorization

### DISCUSSION THREAD

**Discussion by Galaxy3997** [penge/my-notes#428] [Ideas]
I suggest enhancing the sidebar by adding a feature that allows users to categorize their notes. This addition would enable users to organize their notes into different categories, making it easier to sort and find specific pieces of information.
Something like this:
![notesidea](https://github.com/penge/my-notes/assets/139671143/e1f6d537-0d7d-4123-ac7e-ec32034bbed2)

**Comment by penge** [comment]
Hi and thank you for the suggestion!

I am planning to add something similar to version 4 (https://github.com/penge/my-notes/issues/195#issuecomment-1868576538). It will look similar to what you propose, but Category will be also a note.

**Reply by Galaxy3997** [reply]
Oh! Thank you so much! I'm very excited!
